test/pummel/test-timers.js

```diff
@@ -31,7 +31,7 @@ let interval_count = 0;
 clearTimeout(null);
 clearInterval(null);
 
-assert.strictEqual(true, setTimeout instanceof Function);
+assert.strictEqual(setTimeout instanceof Function, true);
```

Comment:
Why not `assert.ok`?

Reply:
`assert.ok()` would be fine here of course, but I'm not terribly bothered by the strict check either.

There are 5 other files in our test suite that use `strictEqual()` for an `instanceof` check.  If there is consensus that changing it to `assert.ok()` would be better, I'd rather change them all at once in a separate PR and add a lint rule.

Reply:
Yeah, that should not block this PR, just asking. :smiley: